system-addon/test/schemas/pings.js

```diff
@@ -63,14 +63,34 @@ const SessionPing = Joi.object().keys(Object.assign({}, baseKeys, {
   session_id: baseKeys.session_id.required(),
   page: baseKeys.page.required(),
   session_duration: Joi.number().integer().required(),
-  action: Joi.valid("activity_stream_session").required()
+  action: Joi.valid("activity_stream_session").required(),
+  perf: Joi.object().keys({
+    // Timestamp of the action perceived by the user to trigger the load
+    // of this page.
+    //
+    // Not required at least for error cases where the
+    // observer event doesn't fire
+    load_trigger_ts: Joi.number().positive(),
```

Comment:
None of the numbers are technically required, though in some ideal world they'd be there every time.  I just suspect that there's going to be some complex code paths where one or another event doesn't fire, and I'd like us to know about them.  That said, I guess we could mark them as required and just see if anything goes wrong.  Thoughts?